cmake/FDBComponents.cmake

```diff
@@ -199,21 +199,6 @@ else()
   target_include_directories(toml11_target SYSTEM INTERFACE ${CMAKE_CURRENT_BINARY_DIR}/toml11/include)
 endif()
 
-################################################################################
-# Coroutine implementation
-################################################################################
-
-set(DEFAULT_COROUTINE_IMPL boost)
-if(WIN32)
-  # boost coroutine not available in windows build environment for now.
-  set(DEFAULT_COROUTINE_IMPL libcoro)
-elseif(NOT APPLE AND NOT USE_ASAN AND CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "^x86")
-  # revert to libcoro for x86 linux while we investigate a performance regression
```

Comment:
Has this changed? Is there still a performance regression?

Reply:
good point. not sure what benchmark can show the regression? @xis19 , before merging, let's find the benchmark and compare performance first.

Reply:
Would circus work? Or it is already retired?